scripts/core/superdesk-api.d.ts

```diff
@@ -62,6 +63,7 @@ declare module 'superdesk-api' {
                 article?: {
                     getActions?(article: IArticle): Promise<Array<IArticleAction>>;
                     getActionsBulk?(articles: Array<IArticle>): Promise<Array<IArticleActionBulk>>;
+                    getExtraActions?(article: IArticle): Promise<Array<IArticleAction>>;
```

Comment:
But there's a difference between those two markForuser: [it's passing true](https://github.com/superdesk/superdesk-client-core/pull/3051/files#diff-ea9bad13dd1bfd970a7ad71b5b1af87dR10) to ignore that the item is locked

Reply:
I'll be fixing a related bug and will take a look at this as well. Ideally, we should change `managed-marked-user.tsx` not to accept a different argument, but take locking into account when passing this argument `superdesk.entities.article.isLocked(article)`. If we made it `superdesk.entities.article.isLocked(article) && article._id !== currentArticleInEditMode` there wouldn't be a difference you mention. Of course, it would require to figure out how to check for the current article in edit mode, but that's an important API that we'll need to add anyway.

Reply:
yep, that would be perfect

Reply:
Fixed in https://github.com/superdesk/superdesk-client-core/pull/3055

Reply:
You meant removing `getExtraActions` and calling `getActions` directly or just include some of the actions from `getActions` inside `getExtraActions`?

Reply:
The former - include everything from `getActions` and drop `getExtraActions` completely since it's the same thing.